README.md

```diff
@@ -1,3 +1,9 @@
+# Archived: This repo is no longer used by flashbots.
+
+This repo is deprecated in favor of the validation API in the flashbots builder.
```

Comment:
Maybe we should add a short note on why we deprecate it
> We are phasing out this repository because we have made the builder repository open source. To avoid the extra work involved in testing two repositories, which increases the chance of errors, we have decided to focus only on verifying the builder repository. The additional effort required to maintain separate changes for the sole purpose of separation is outweighed by the extra testing needed.

Reply:
+1

Reply:
```suggestion
This repo is deprecated in favor of the validation API in the flashbots builder.

We are phasing out this repository because we have made the builder repository open source. To avoid the extra work involved in testing two repositories, which increases the chance of errors, we have decided to focus only on verifying the builder repository. The additional effort required to maintain separate changes for the sole purpose of separation is outweighed by the extra testing needed.
```